themes/app/templates/Includes/HomePageTop.ss

```diff
@@ -0,0 +1,10 @@
+<div class="container homepage__top">
+        <div class="row">
+            <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12 col-12">
```

Comment:
Same here, can remove `col-xl-6` and `col-12`.

Actually I just remembered, try removing `col-sm-12` I think bootstrap should be smart enough to recognize it will go into a full width layout when it hits lower break points. 

So you can probably just refactor this to `col-md-6` and see what happens :)